lib/cartopy/_crs.pyx

```diff
@@ -417,8 +419,8 @@ cdef class CRS:
         -------
             ut, vt: The transformed vector components.
 
-        Notes
-        -----
+        Note
```

Comment:
> napoleon supports both spellings

True, but it does ***not*** say that 'Note' is an alias for 'Notes' (as it does for e.g. 'Return'/'Returns', and others)

> unnecessary churn

I can only say that when I tried it, a "Notes" section did *not* produce a highlighted box, whereas a "Note" section does. So that is why I changed them all ***back*** from what had been done in #987 (maybe not all cases, but certainly most ?).
As these were originally written as pure Sphinx RST, those sections were originally styled as highlighted boxes, and in several cases that still seems highly desirable for the clarity of content.

The official numpy documentation style notes (https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) make no mention of ***any*** 'Note' or 'Notes' section, but instead suggest that you should use a Sphinx "..note:".  So I assume it must be a specifically napoleon usage, but the napoleon docs (which you reference) don't make it fully clear.

TBH I am more interested in what works than what is "right" here, because I don't think the documentation is sufficiently complete and definitive to be sure of all the facts, sadly.  
So if someone points out definite mis-use then that is good cause to change, but otherwise I think we must accept that "what works" is still the final arbiter for a lot of these style points.

Reply:
Every entry in the Sections I linked for numpydoc is the name of a heading; see [the example file](https://github.com/numpy/numpy/blob/c90d7c94fd2077d0beca48fa89a423da2b0bb663/doc/example.py#L86) as well. But if it doesn't look good, then that trumps that.

Reply:
Arguably where this was the desired effect, we could have reverted to "..note::", as it is a standard RST directive, but I guess the use of napoleon makes sense for now.
In principle, a "..note::" section can appear within any other (RST) section, so it might be a better fit.  However, I have found that including a note within an argument description confuses napoleon, so in fact I don't think it adds any utility.